### PR TITLE
transform-geojson: Accept `url_template` and `layer_options` metadata

### PR DESCRIPTION
**Pull request by gnestor**
Closes: https://github.com/nteract/nteract/issues/1574

This currently depends on a pending PR on ipython: https://github.com/ipython/ipython/pull/10404

Usage:

```py
from IPython.display import GeoJSON

GeoJSON(data={
    "type": "Feature",
    "geometry": {
        "type": "Point",
        "coordinates": [11.8, -45.04]
    }
}, url_template="http://s3-eu-west-1.amazonaws.com/whereonmars.cartodb.net/{basemap_id}/{z}/{x}/{y}.png", 
layer_options={
    "basemap_id": "mola-color",
    "attribution" : "NASA/MOLA",
    "tms": True,
    "minZoom" : 0,
    "maxZoom" : 6
})
```

image

### CODE REVIEW

**Comment by gnestor** [comment]
@rgbkrk Can you renew the mapbox access token?

**Comment by rgbkrk** [comment]
> Can you renew the mapbox access token?

We should probably switch to openstreetmap for the defaults.

**Comment by rgbkrk** [comment]
Also I think I took the token from sample docs originally, can't find another one without creating one that would be associated to a mapbox account.

**Comment by gnestor** [comment]
No more luma-based theme 😢️

File: packages/transform-geojson/src/index.js [hunk 1]
```diff
@@ -85,28 +70,36 @@ export class GeoJSONTransform extends React.Component {
...
       const geoJSON = this.props.data;
-
       this.map.removeLayer(this.geoJSONLayer);
       this.geoJSONLayer = L.geoJson(geoJSON).addTo(this.map);
       this.map.fitBounds(this.geoJSONLayer.getBounds());
     }
   }
 
+  getTileLayer(): TileLayer {
+    const metadata = this.props.metadata.toJSON();
```
Comment (AlJohri) [inline packages/transform-geojson/src/index.js]: @gnestor I'm getting `TypeError: Cannot read property 'toJSON' of undefined` here.

Comment (gnestor) [reply packages/transform-geojson/src/index.js]: Just pushed a commit to fix that 👍

**Comment by gnestor** [comment]
I think this is ready to merge!

**Comment by rgbkrk** [comment]
Thanks @gnestor!

**Pull request merged**